src/Components/SingleArticle.jsx

```diff
@@ -34,19 +35,24 @@ export default function SingleArticle() {
   }
 
   return (
-    <li className="card bg-base-100 w-96 shadow-xl p-4 m-10 w-25">
-      <p>by: {singleArticle.author}</p>
```

Comment:
on line 30 and 34 above the loading and error aren't being returned so we dont see on screen when the internet is slow